extensions/amp-accordion/1.0/test/test-amp-accordion.js

```diff
@@ -516,6 +517,94 @@ describes.realWin(
       });
     });
 
+    describe('display locking', () => {
+      let defaultCssSupports;
+      let defaultBeforeMatch;
+
+      beforeEach(async () => {
+        toggleExperiment(win, 'amp-accordion-display-locking', true);
+        element = html`
+          <amp-accordion>
+            <section>
+              <h2>Section 1</h2>
+              <div>Puppies are cute.</div>
+            </section>
+            <section expanded>
+              <h2>Section 2</h2>
+              <div>Kittens are furry.</div>
+            </section>
+            <section expanded>
+              <h2>Section 3</h2>
+              <div>Elephants have great memory.</div>
+            </section>
+          </amp-accordion>
+        `;
+        defaultCssSupports = win.CSS.supports;
+        defaultBeforeMatch = win.document.body.onbeforematch;
+      });
+
+      afterEach(() => {
+        win.CSS.supports = defaultCssSupports;
+        win.document.body.onbeforematch = defaultBeforeMatch;
+      });
+
+      it('should expand collpased section with beforematch event', async () => {
+        win.document.body.appendChild(element);
+        win.CSS.supports = () => true;
+        win.document.body.onbeforematch = null;
+        await element.build();
+
+        const section1 = element.children[0];
+        const content1 = section1.lastElementChild;
+
+        expect(win.AMP.isExperimentOn('amp-accordion-display-locking')).to.be
```

Comment:
Do we need this expect statement if we know we turn on the experiment in the `beforeEach` block?

Reply:
I don't think it hurts?

Reply:
I don't think it helps. IMO we should remove so the tests can concisely add confidence. But it's up to you.